Resources/config/imagine.xml

```diff
@@ -28,6 +28,18 @@
 
         <parameter key="liip_imagine.mozjpeg.binary">/opt/mozjpeg/bin/cjpeg</parameter>
 
+        <!-- cwebp parameters -->
+
+        <parameter key="liip_imagine.cwebp.binary">/usr/bin/cwebp</parameter>
+        <parameter key="liip_imagine.cwebp.near_lossless">100</parameter>
+        <parameter key="liip_imagine.cwebp.q">75</parameter>
+        <parameter key="liip_imagine.cwebp.alpha_q">100</parameter>
+        <parameter key="liip_imagine.cwebp.m">4</parameter>
+        <parameter key="liip_imagine.cwebp.alpha_filter">fast</parameter>
+        <parameter key="liip_imagine.cwebp.alpha_method">1</parameter>
+        <parameter key="liip_imagine.cwebp.exact">false</parameter>
+        <parameter key="liip_imagine.cwebp.metadata">none</parameter>
```

Comment:
this is consistent with the other post processors, so lets do it this way for now. but i think it would be much better if this was using the symfony DI configuration system... something for the next major version.

Reply:
Yes, I'm not proud of this either. I was just doing same as others post-processors to remain consistent.